common/tests/refcount_ut/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(theseTestsName refcount_ut)
+
+set(${theseTestsName}_test_files
+${theseTestsName}.c
+)
+
+set(${theseTestsName}_c_files
+	some_refcount_impl.c
+)
+
+set(${theseTestsName}_h_files
+	some_refcount_impl.h
+)
+
```

Comment:
Likely need to add include directory to either WIN32/LINUX depending on platform until you fix refcount to just be a "common" module that uses the PAL interlocked.